crates/starknet/src/starknet/mod.rs

```diff
@@ -275,6 +275,12 @@ impl Starknet {
     ) -> Result<(TransactionHash, ClassHash)> {
         add_declare_transaction::add_declare_transaction_v2(self, declare_transaction)
     }
+
+    /// returning the block number that will be added, ie. the most recent accepted block number
+    pub fn block_number(&self) -> Result<BlockNumber> {
```

Comment:
Does it need to be `Result`? Can't it be just `BlockNumber` if it's never going to fail?

Reply:
cc: @marioiordanov

Reply:
[[11db370](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/77/commits/11db37013e07aa8c137653a0783f194ac99bedf7)](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/77/commits/11db37013e07aa8c137653a0783f194ac99bedf7)